dev/CommonStyles/AppBarButton_themeresources.xaml

```diff
@@ -344,7 +344,12 @@
                                 BorderThickness="{TemplateBinding BorderThickness}"
                                 contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                                 contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
-                                Control.IsTemplateFocusTarget="True" />
+                                Control.IsTemplateFocusTarget="True">
+
+                            <contract7Present:Border.BackgroundTransition>
+                                <contract7Present:BrushTransition Duration="0:0:0.083" />
```

Comment:
Can this use the named resource?

Reply:
I wish. :/ The short answer is no. BrushTransition.Duration is not of type Duration (naturally) but is instead a TimeSpan and there's a markup issue with those. Check out bug 30603142.